calcom/cal.com
# feat: Twipla app analytics added

This PR is to add Twipla analytics in the app store

fixes: #14758 

loom video:
https://www.loom.com/share/2e97fea489044271b7f6f3fe03ccc52a

## Type of change



- New feature (non-breaking change which adds functionality)


## How should this be tested?

- Add Twipla analytics application to an event type.
- Open up the event page and it should automatically track it.

## Mandatory Tasks
